config/initializers/feature_flags.rb

```diff
@@ -0,0 +1,11 @@
+FEATURES = {
+  basic_auth: {
+    enabled:  Rails.env.production? || (!ENV['BASIC_AUTH_FORCE'].blank? && ENV['BASIC_AUTH_FORCE'] != 'false'),
```

Comment:
Does `BASIC_AUTH_FORCE` feel like it goes beyond turning basic auth off or on?

Could we commute the contents of the `()` expression to `ENV['BASIC_AUTH_FORCE'] == 'true'`? ENV only deals in strings, and `ENV['nonexistent']` will return `nil`.

Reply:
True, I've just seen people change an env var to 'false' to turn it off too many times.

Reply:
== 'true' should cope with that, amending